Main.java

```diff
@@ -0,0 +1,54 @@
+import java.io.BufferedReader;
+import java.io.FileReader;
+
+public class Main {
+
+	public static void main(String[] args) {
```

Comment:
It would have been much easier to review if you had comments explaining your code.